pkg/ciliumenvoyconfig/cec_manager.go

```diff
@@ -100,7 +100,7 @@ func (r *cecManager) addCiliumEnvoyConfig(cecObjectMeta metav1.ObjectMeta, cecSp
 		// the bpf maps are not updated with the new proxy ports either. Move from the
```

Comment:
Random thought, there's a watchdog job that also checks if the expected tc hooks are in place for endpoints periodically, failing that it triggers a full datapath loader `Reinitialize()`. Wonder if we need to do that, or if instead we can just have that trigger this to regenerate all endpoints. 

Might be one less thing starting redundant endpoint regenerates.

Reply:
Ooh, good observation. I'll take a look.

Reply:
Took a quick look, the tc watchdog calls `Orchestrator.Reinitiaize()`, whereas endpoint regeneration calls `Orchestrator.ReloadDatapath(ep)`. Also, the periodic regeneration is without BPF compilation, whereas the watchdog triggers a full bpf regen. So, different context.